app/assets/javascripts/angular/services/graph_refresher.js

```diff
@@ -47,12 +47,12 @@ angular.module("Prometheus.services").factory('GraphRefresher',
         });
         return deferred.promise;
       },
-      graphite: function(idx, expression, server, expressionID, endTime, range, step) {
+      graphite: function(idx, expression, server, expressionID, startTime, endTime, step) {
         var deferred = $q.defer();
         $http.get(URLGenerator(server.url, 'render', $scope.vars), {
           params: {
             target: expression,
-            from: GraphiteTimeConverter.graphiteFrom(range, endTime),
+            from: GraphiteTimeConverter.graphiteUntil(startTime),
```

Comment:
Ah, I guess I get it - but is the `graphiteUntil` function still named well then?